### PR TITLE
 Overview page: Grid widget: confusion when some phases in watts and others in kilowatts

### PR DESCRIPTION
**Pull request by DanielMcInnes**
'Small' AcWidgets no longer hide the total power, as there is enough space to show it.
'ExtraSmall' and 'Small' AcWidgets now display units for each phase, eg 'kW', 'W'.

### CODE REVIEW

**Comment by blammit** [comment]
Can you improve the commit's first line so it is less generic than "fix bug in X"? E.g. "Show per-unit phase in overview AC widget" or "Change display of units and total quantity in overview AC widget". Also add a reference to the issue number.

**Comment by DanielMcInnes** [comment]
> Can you improve the commit's first line so it is less generic than "fix bug in X"? E.g. "Show per-unit phase in overview AC widget" or "Change display of units and total quantity in overview AC widget". Also add a reference to the issue number.

done

**Comment by DanielMcInnes** [comment]
Serj commented:

Hi, thank you for sharing this. One problem i see is that text may become unreadable on target screen at that dimension (smallest we currently have is 12px) However changing from Museo to Roboto font gives us more vertical space, as Roboto is more condensed. In scenarios where we have multiple energy sources - 5 containers on the left, the totals for 3 phases wont fit, and there can be two sources with 3 phases and 5 blocks to show (super yacht scenario).  I would say, if we can read it on GX target screen then do not mind to go with proposal you shared. Taking in consideration all corner cases.

**Comment by DanielMcInnes** [comment]
I can't test on the device with 5 widgets on the left, but on the desktop, a 5" display looks ok:
image

**Review by chriadam — APPROVED** [review]

File: components/ThreePhaseDisplay.qml [hunk 1]
```diff
@@ -75,7 +75,6 @@ Flow {
 						: 0
 				dataObject: model
 				font.pixelSize: phaseLabel.font.pixelSize
-				unitVisible: root.widgetSize >= VenusOS.OverviewWidget_Size_M
```
Comment (blammit) [inline components/ThreePhaseDisplay.qml:78]: Can you remove `unitVisible` from QuantityLabel? With this change, it will no longer be used.

**Review by blammit — APPROVED** [review]
LGTM, minor comment about clean-up.

File: components/ThreePhaseDisplay.qml (hunk 1 above)
Comment (DanielMcInnes) [reply components/ThreePhaseDisplay.qml:78]: Done
